backend/functions/src/triggers/on-create-bet.ts

```diff
@@ -153,7 +153,7 @@ const addBetToFollowersFeeds = async (
   contract: Contract,
   bet: Bet
 ) => {
-  if (bettor.followerCountCached <= 0 || contract.mechanism === 'dpm-2') return
```

Comment:
This was handy for exiting early, would be nice to have this in so we don't waste time querying peoples' bets who don't have followers.